frontend/src/components/Rating/Review.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { FaStar } from "react-icons/fa";
+import { useState } from "react";
+import "./Review.css";
+function Review({ mealId }) {
+  const [rating, setRating] = useState(null);
+  const [hover, setHover] = useState(null);
+  let currentRating = 1;
+  const [newReview, setNewReview] = useState({
+    stars: "",
+    meal_id: "",
+    create_date: "",
+    title: "",
+  });
+
+  const clickHandler = (event) => {
+    event.preventDefault();
+
+    const starValue = Number(event.target.value);
+    setRating(starValue);
+
+    const meal_id = mealId;
```

Comment:
Is there a special reason why to reassign the `mealId` to a new variable? I think you can use `mealId` as is, no?

Reply:
I don't know why!! I changed.